signer/contentsignaturepki/x509.go

```diff
@@ -37,6 +37,10 @@ func (s *ContentSigner) findAndSetEE(conf signer.Configuration) (err error) {
 		err = errors.Wrapf(err, "found suitable end-entity labeled %q in database but not in hsm", s.eeLabel)
 		return
 	}
+	if len(s.PublicKey) < 50 {
```

Comment:
Should we check this in `GetKeys` / will we ever use a public key < 50 bytes?

Reply:
makes sense, moved it there